pkgs/applications/virtualization/bochs/default.nix

```diff
@@ -23,21 +23,21 @@ stdenv.mkDerivation rec {
   version = "2.7";
 
   src = fetchurl {
-    url = "mirror://sourceforge/project/bochs/bochs/${version}/bochs-${version}.tar.gz";
+    url = "mirror://sourceforge/project/${pname}/${pname}/${version}/${pname}-${version}.tar.gz";
```

Comment:
This makes getting the original URL very hard because someone needs to copy all the variables together. There is no actual benefit of re-using pname at every place the string is used and when the project moves/renames there usually needs to be more done than just changing pname.